about.html

```diff
@@ -94,10 +98,11 @@ <h2>Partners</h2>
         </section>
 
         <section class="footer">
```

Comment:
You're doing a great job refining your project! Here's a helpful review for your consideration:

- [ Optional] In the `about.html` page, it's important to note that the partner section in the footer should be visible in the mobile version, but hidden in the desktop version. To align with the design's intent, please make the necessary adjustments in your code.

![image](https://github.com/amiralda/haitian-Otaku-Website/assets/55840999/9259a985-0abd-40ba-852e-91a9237216db)

By addressing this detail, you'll ensure that your project adheres closely to the design specifications. Keep up the excellent work! 🌟📝🚀